ec2utils/ec2utilsbase/lib/ec2utils/ec2utilsutils.py

```diff
@@ -75,10 +74,10 @@ def find_images_by_name(images, image_name):
     """Return a list of images that match the given name."""
     matching_images = []
     for image in images:
-        if not image['Name']:
+        if not image.get('Name'):
             print(_no_name_warning(image))
             continue
-        if image_name == image['Name']:
+        if image_name == image.get('Name'):
```

Comment:
When the flow of control gets here we know the 'Name' key exists thus we can use "direct" access ['Name'] dito for other occurrences.